mailet/api/src/main/java/org/apache/mailet/Mailet.java

```diff
@@ -127,14 +129,19 @@ public interface Mailet {
     /**
      * Returns information about the mailet, such as author, version and
      * copyright.
+     * <p>
+     * By default, this method returns an empty string. Override this method
+     * to have it return a meaningful value.
      *
      * @return the Mailet information (as a plain text string)
      */
-    String getMailetInfo();
+    default String getMailetInfo() {
+        return "";
```

Comment:
Maybe we could default to getClass().getSimpleName()?

(ok that is a behavioural change... So maybe in another PR, but providing sane defaults for this would look good to me. One less thing to care about)

Reply:
MailetInfo is not the same as the mailet name, the javadoc says its about 

> author, version and copyright 

but using `SimpleName` is an interesting idea for the next PR ;)

Reply:
I may have had overlooked this sorry.